src/containers/ProjectContainer.js

```diff
@@ -124,18 +124,32 @@ const ProjectContainer = React.createClass({
 
   fetchPullRequests() {
     let { gitHubToken, selectedRepo } = getSettings();
-    return $.ajax({
-      url: `https://api.github.com/repos/${selectedRepo}/pulls?state=open&access_token=${gitHubToken}`,
-      method: 'GET'
-    }).then(data =>
-      _.map(data, pullRequest => (
-        {
-          id: pullRequest.id,
-          url: pullRequest.html_url,
-          commitsUrl: pullRequest.commits_url
-        }
-      ))
-    ).fail(() => this.setState({ error: true }));
+    return new Promise(resolve => {
+      if (selectedRepo !== null && selectedRepo !== undefined) {
+        $.ajax({
+          url: `https://api.github.com/repos/${selectedRepo}/pulls?state=open&access_token=${gitHubToken}`,
+          method: 'GET'
+        }).then(data =>
+          resolve(
+            _.map(data, pullRequest => (
+              {
+                id: pullRequest.id,
+                url: pullRequest.html_url,
+                commitsUrl: pullRequest.commits_url
+              }
+            ))
+          )
+        ).fail(() => { 
+          this.setState({ error: true }); 
+          resolve(); 
+        });
+      } else {
+        resolve();
+      }
+    });
+
+  
+
```

Comment:
Lines 150-152

You can take those lines out and then ammend your git commit and force push the branch back up and it will update the PR.